Library/ViewModels/DiscoveryViewModel.swift

```diff
@@ -39,6 +39,12 @@ public protocol DiscoveryViewModelOutputs {
   /// Emits an array of sorts that should be used to configure the sort pager controller.
   var configureSortPager: Signal<[DiscoveryParams.Sort], NoError> { get }
 
+  /// Emits a boolean that determines if the discovery pages view is hidden.
+  var discoveryPagesViewHidden: Signal<Bool, NoError> { get }
+
+  /// Emits a boolean that determines if the live stream discovery view is hidden.
+  var liveStreamDiscoveryViewHidden: Signal<Bool, NoError> { get }
```

Comment:
so the livestream discovery controller is an embedded controller in `DiscoveryViewController`, which means the discovery pages are its siblings. this vm will now take care of hiding and showing them appropriately.